.ideainspect

```diff
@@ -21,3 +15,10 @@ resultdir: target/inspection-results
 
 # IDEA project root directory containing the ".idea" directory
 # rootdir: .
+
+# Library folder containing idea.properties More info at:
+# http://tools.android.com/tech-docs/configuration
+# lib: ~/Library/Preferences/AndroidStudio2.1/
```

Comment:
I'll probably just substitute the whole thing for the absolute path of idea.properties.